### PR TITLE
Allowing backup puzzles as extra during Multi-Blind

### PR DESCRIPTION
**Pull request by hanwu85**
Addressing issue #766 .

Let's discuss if there are loophole or grey area in these wordings and over how many cubes should we recommend?

### CODE REVIEW

**Review by Rouxles — CHANGES_REQUESTED** [review]
> - H1a+) [ADDITION] The competitors may choose to bring 1 or 2 extra puzzles as backup in case of incidents found during the attempt (e.g. duplicate scrambled puzzles). Attempts higher than 6 puzzles are highly recommendated to bring extra puzzles.

For this, would something like `The competitor may choose to bring 1 or 2 extra puzzles as backup in case of duplicate scrambles found during the attempt` be better?

For this, `Attempts higher than 6 puzzles are highly recommendated to bring extra puzzles.`, would it be better to put it as a sub-guideline (as in it's under H1a+)?

**Comment by hanwu85** [comment]
My idea is the following,  if mis-scramble cubes, cubes with logos (or other illegal cubes) are found, the judges or delegates can swap the cubes.  If we say "duplicate cubes", those are not allowed to be swapped.   So that's why I think we should brainstorm to find loopholes.

What do you mean by sub-guidelines?

**Comment by Ivan-Brigidano** [comment]
Why only 1 or 2 cubes and not was many as the competitor consider?

**Comment by hanwu85** [comment]
> Why only 1 or 2 cubes and not as many as the competitor consider?

That's also my question, I think as many as they want will cause other issues like deliberately wasting competition resource.  
How many is considered enough is open for discussion.
 
I would also like to know how often MBLD incidents happen in competitions.

**Comment by Ivan-Brigidano** [comment]
Maybe something like one cube for backup every 6 could work

**Comment by Laura-O** [comment]
> Why only 1 or 2 cubes and not was many as the competitor consider?

And in case one of them is illegal, maybe also a replacement for them?

Seriously, has one of you ever organized a competition holding MultiBLD?
MultiBLD already is an organizational nightmare for several reasons. The more cubes competitors attempt and the more cubes you have to deal with, the worse it gets.

Apart from finding time to scramble puzzles, the main problem is that you have to find a place where you can collect the puzzles, scramble them and then find a way to somehow carry them to the place where competitors will do their attempts. All of this needs to be done while making sure that competitors do not see a scrambled cube.
We have scrambled MultiBLD in hallways, cellars or stuffy backstage rooms, sometimes standing there for 45 minutes because there was no space for a chair. I guess you can imagine that there would also be not much room for additional cubes.

The suggested change makes this worse and even creates more potential incidents:
1. You would have to keep 2 piles of cubes for each competitor (one for regular, one for backup cubes) and make sure they do not get mixed up, which basically means that you would have to print two scoresheets for each competitor.
2. You have to keep the scrambled backup cubes at a place where competitors can't see them until the round ends. At most competitions, it's already difficult to find a place to scramble them and keep them until the round starts. The backup cubes would have to be placed somewhere where they are accessible during the round but still hidden from the competitors.
3. What happens when regular cubes and backup cubes get mixed? With a large number of regular and backup cubes that's not unlikely.

**Comment by Ivan-Brigidano** [comment]
> Seriously, has one of you ever organized a competition holding MultiBLD?

Yes and I'm aware of how difficult is (sometimes) to hold it.

I think that a few cubes are reasonable but no unlimited. I know this could make holding the event even more difficult and cause some new problems. I think this is reasonable if backup cube are not in a different room but in the same one covered with and opaque cloth.

**Comment by Laura-O** [comment]
> I think that a few cubes are reasonable but no unlimited. I know this could make holding the event even more difficult and cause some new problems. I think this is reasonable if backup cube are not in a different room but in the same one covered with and opaque cloth.

This still means that you need to place them somewhere and you need more covers.

I can imagine that this works well at smaller competitions with a lot of space/rooms. However, in areas where competitions usually reach the competitor limit and many competitors are allrounders, who frequently do MultiBLD, this can be a serious problem.
I can remember several venues we used where putting 20 additional piles of cubes somewhere (and assuring scramble secrecy) would have been nearly possible. Not to mention the additional time it would have needed to scramble them.

**Comment by Ivan-Brigidano** [comment]
> I can imagine that this works well at smaller competitions with a lot of space/rooms. However, in areas where competitions usually reach the competitor limit and many competitors are allrounders, who frequently do MultiBLD, this can be a serious problem.

Yes, there could be several issues in some competitions. Maybe another option could be allowing to re-scramble the cube that is duplicated.

**Comment by EdHollingdale** [comment]
I agree fully with Laura. What problem are we trying to fix here? It feels like you are creating the possibility of a logistical nightmare to make a rare incident marginally better. The instances this would fix are when there is a duplicate scramble, or an illegal cube. Both of these can be fixed far better by encouraging a scramble checker - which also solves additional problems.

As an example, the other weekend we had 21 multi competitors, including a few with large attempts. It is already hard enough managing the cubes (making sure they stay together, are scrambled correctly, are hidden from view, are brought out to the solving stations). Let alone managing an extra ~40 back-up cubes which would need to be handled completely independently so as not to be mixed up with the main cubes, still need to be scrambled and moved into the main room. And now I need to get an extra 21 opaque cloths to cover them? All for the possibility that our most trusted scrambler and a delegate who was checking their work made a mistake? 

And even if there is a duplicate, how does having a backup cube help? It's not like the competitor is unaffected - they still have to notice the duplicate, call a delegate over, wait while the delegate confirms this, and then wait while the delegate works out under which of the 21 opaque cloths their additional cube is placed, then start memorising the new cube once the delegate has brought it over.

> Maybe another option could be allowing to re-scramble the cube that is duplicated.

Good idea! No regulations change needed!

File: wca-guidelines.md [hunk 1]
```diff
@@ -222,6 +222,8 @@ To be more informative, each Guideline is classified using one of the following
 
 ## <article-H><multiple-blindfolded><multipleblindfoldedsolving> Article H: Multiple Blindfolded Solving
 
+- H1a+) [ADDITION] The competitors may choose to bring 1 or 2 extra puzzles as backup in case of incidents found during the attempt (e.g. duplicate scrambled puzzles). Attempts higher than 6 puzzles are highly recommendated to bring extra puzzles.
```
Comment (pedrosino) [inline wca-guidelines.md:225]: To be more in line with the rest of the text, I'd suggest to change "The competitors" to simply "Competitors" or "The competitor" (same as H1a)
Also not sure if the "1 or 2" is necessary. Competitors trying 40 cubes may want to have more backups, as the risk of incidents is higher.

Also, recommendated -> recommended

File: wca-guidelines.md [hunk 2]
```diff
@@ -222,6 +222,8 @@ To be more informative, each Guideline is classified using one of the following
...
+- H1a++) [ADDITION] The extra puzzles should be scrambled together with the intended puzzles but kept away from the competitors untill incidents are found.
```
Comment (pedrosino) [inline wca-guidelines.md:226]: Suggestion:
"The extra puzzles must be scrambled together with the intended ones but kept away from the competitor. Extra puzzles must only be used if any incident is found during the attempt."

File: wca-guidelines.md (hunk 1 above)
Comment (xsrvmy) [inline wca-guidelines.md:225]: Another language issues I see here: in the last sentence, "attempts" cannot bring extra puzzles.
Maybe "For attempts with more than 6 puzzles, extra puzzles are highly recommended." or "Competitors attempting more than 6 puzzles should bring extra puzzles."

**Comment by lgarron** [comment]
> > Maybe another option could be allowing to re-scramble the cube that is duplicated.
> 
> Good idea! No regulations change needed!

Based on the feedback, the WRC has agreed that this is an appropriate way to handle it.

**Pull request closed without merging**